unevendream/zh/meta.ini

```
@@ -1,4 +1,4 @@
 [Language]
 Name=简体中文
-Description=Localization by Monkey233 and ynoProjectCN(temporary)
-Term=语言
\ No newline at end of file
+Description=Localization by Moucky233 and ynoProjectCN(temporary)

```

```suggestion
Description=Localization by Moucky233 and YNOprojectCN translation team (temporary name)
```
Might be a bit more formal this way